opal-core-api/src/main/java/org/obiba/opal/core/domain/AnalysisResultWrapper.java

```diff
@@ -0,0 +1,44 @@
+package org.obiba.opal.core.domain;
+
+import com.google.common.collect.Lists;
+import java.util.List;
+import org.obiba.opal.spi.analysis.AnalysisResult;
+
+public class AnalysisResultWrapper implements HasUniqueProperties {
```

Comment:
No sure *Wrapper is a good name. Similar classes starts with Opal instead, so it would be OpalAnalysisResult.